CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## [0.5.1] - 2022-05-26
```

Comment:
add the link as following https://github.com/4paradigm/OpenMLDB/blob/1cc64d0e294f1efdf1d85594a5e2e38063da5172/CHANGELOG.md?plain=1#L240